packages/forms-web-app/src/views/eligibility/householder-planning-permission.njk

```diff
@@ -6,8 +6,13 @@
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 
+{% set title = "Did you apply for householder planning permission? - Eligibility - Appeal a householder planning decision - GOV.UK" %}
+
 {% block pageTitle %}
-  Did you apply for householder planning permission? - Eligibility - Appeal a householder planning decision - GOV.UK
+  {% if errors %}
```

Comment:
could this pattern be applied to the overall page template?

Reply:
I don't think so because we can't assume that this behaviour will be required in every page.